File: javascript-closure-angular-client/API/Client/SimpleAddress.js
goog.provide('API.Client.SimpleAddress');

/**
 * @record
 */
API.Client.SimpleAddress = function() {}

/**
 * Address
 * @type {!string}
 * @export
 */
API.Client.SimpleAddress.prototype.line1;

/**
 * Number
 * @type {!string}
 * @export
 */
API.Client.SimpleAddress.prototype.line2;

/**
 * District
 * @type {!string}
 * @export
 */
API.Client.SimpleAddress.prototype.line3;

/**
 * @type {!string}
 * @export
 */
API.Client.SimpleAddress.prototype.city;

/**
 * Brazilian Zip Code
 * @type {!string}
 * @export
 */
API.Client.SimpleAddress.prototype.zipcode;

/**
 * @type {!API.Client.StateEnum}
 * @export
 */
API.Client.SimpleAddress.prototype.state;

/**
 * Use ISO 3166-1 alpha-3 codes
 * @type {!string}
 * @export
 */
API.Client.SimpleAddress.prototype.country;

File: async-scala-client/src/main/scala/io/swagger/client/model/SimpleAddress.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class SimpleAddress (
  line1: String,  // Address
line2: String,  // Number
line3: String,  // District
city: Option[String],
zipcode: String,  // Brazilian Zip Code
state: Option[StateEnum],
country: String  // Use ISO 3166-1 alpha-3 codes
)
